Makefile

```diff
@@ -94,4 +94,26 @@ clean: ## Clean the project
 
 .PHONY: pre-commit-hook
 pre-commit-hook: ## Install pre-commit hook in .git/hooks
-	/usr/bin/cp -fa .githooks/pre-commit .git/hooks/pre-commit
\ No newline at end of file
+	/usr/bin/cp -fa .githooks/pre-commit .git/hooks/pre-commit
+
+.PHONY: local-appcat-development
```

Comment:
Can you add some comments here what these targets do? It's not clear. Also this has nothing to do with this PR but it's ok for me as long as this is documented what it does and why it exists otherwise nobody will know about it let alone use it.

Reply:
I removed that logic as even I stopped using that